cmd/system-probe/api/module/loader.go

```diff
@@ -45,6 +46,10 @@ type loader struct {
 // * Initialization using the provided Factory;
 // * Registering the HTTP endpoints of each module;
 func Register(cfg *config.Config, httpMux *mux.Router, factories []Factory) error {
+	if err := driver.Init(cfg); err != nil {
```

Comment:
if `driver.Init()` fails, should it fail to load the whole module?  Since the module can run w/o the driver?

Reply:
We don't want to bring down all of system-probe if a driver fails for some reason - only want to fail the parts of system-probe that need the driver.